integration_tests/src/testcases/core.test.ts

```diff
@@ -923,6 +962,14 @@ describe('Core', () => {
     await checkExchangeRate(context);
   });
 
+  it('query redemption rate', async () => {
+    const rate = await context.redemptionAdapterClient.queryRedemptionRate({
+      denom: `factory/${context.tokenContractClient.contractAddress}/udatom`,
+    });
+    expect(rate.redemption_rate).toEqual('1');
+    expect(Date.now() / 1000 - rate.update_time).toBeLessThan(5);
```

Comment:
It would be great to compare redemption rate against our exchange rate.

Reply:
they are the same, or do you want to make sure are the same?

Reply:
yes, to make sure they are the same.